### DISCUSSION TITLE
help please

### DISCUSSION THREAD

**Discussion by ah-ha-admin** [termux/termux-app#3335] [Q&A] [answered]
i wanted to remove npm because i changed something in config file that made a bug so i used : rm -rf /data/data/com.termux/files/usr/bin/npm
now i'm having issues , how to fix this ? 
![Screenshot_20230409-123327_Termux](https://user-images.githubusercontent.com/118419708/230785452-19812094-6f23-4868-b8d5-a18297e0a4c7.jpg)

**Comment by sylirre** [comment, marked as answer]
Its kinda strange that you've decided to delete a binary after breaking something in configuration file. I think it should be obvious that deleting and reinstalling a package would not solve issue with configuration file because latter is not removed, isn't it?

Another point is that one should not attempt to `rm` something manually in `$PREFIX` without fully understanding the consequences.
